generator/src/main/scala/grpcgateway/generators/GatewayGenerator.scala

```diff
@@ -52,7 +52,8 @@ object GatewayGenerator extends protocbridge.ProtocCodeGenerator with Descriptor
         "import _root_.com.trueaccord.scalapb.json.JsonFormat",
         "import _root_.grpcgateway.handlers.GrpcGatewayHandler",
         "import _root_.io.grpc.ManagedChannel",
-        "import _root_.io.netty.handler.codec.http.{HttpMethod, QueryStringDecoder}"
+        "import _root_.io.netty.handler.codec.http.{HttpMethod, QueryStringDecoder}",
+        "import org.json4s.JsonAST._"
```

Comment:
I'd like to avoid any direct dependency to a JSON library (other than scalp-json4s) ...

Reply:
Can you explain why? It's not conveniet to work with primitive strings. I can filter out request that have bad json string right in GatewayHandler. scalap-json4s is compiled dependency of runtime

Reply:
`json4s` is the library used by `scalapb-json4s`. The gateway doesn't depend directly on `json4s` (only on `scalp-json4s`). If scalaPB were to choose another json library the gateway should still work but if it starts using `json4s` directly it's no longer the case.

Reply:
ok, i get it. I've removed it